src/openrct2/platform/Posix.cpp

```diff
@@ -98,7 +98,7 @@ void platform_get_time_local(rct2_time* out_time)
 static size_t platform_utf8_to_multibyte(const utf8* path, char* buffer, size_t buffer_size)
 {
     wchar_t* wpath = utf8_to_widechar(path);
-    setlocale(LC_CTYPE, "UTF-8");
+    setlocale(LC_CTYPE, "en_US.UTF-8");
```

Comment:
This will fail if the `en_US` locale is absent, right? Not unheard of, unfortunately. (In the past, I only had `en_UK` and `nl_NL` locales installed to save a bit of space...)

Reply:
Sure, I was waiting for feedback on this. I just found that this fixed it - so an improvement on before but not yet good enough.

Any ideas?

Reply:
We could go by the return value, if its absent it will return NULL otherwise the locale assigned.